ubuntucleaner/janitor/aptcache_plugin.py

```diff
@@ -15,13 +15,14 @@ class AptCachePlugin(JanitorCachePlugin):
 
     def clean_cruft(self, cruft_list=[], parent=None):
         for index, cruft in enumerate(cruft_list):
-            log.debug('Cleaning...%s' % cruft.get_name())
-            result = proxy.delete_apt_cache_file(cruft.get_name())
-
-            if bool(result) == False:
-                self.emit('clean_error', cruft.get_name())
-                break
-            else:
-                self.emit('object_cleaned', cruft, index + 1)
+            if cruft:
```

Comment:
What was the case that this was failing?

Reply:
This was an attempted bugfix since the aptcache plugin crashes in elementaryOS.

Reply:
Then this is not a bugfix, is implementing/duplicating code instead of actually fixing the bug which causes the aptcache plugin to fail in elementaryOS.